File: backend/user-service/app/models/blood_pressure.ts
import { BaseModel, column } from '@adonisjs/lucid/orm'

export default class BloodPressure extends BaseModel {
  static table = 'blood_pressure'

  @column({ isPrimary: true })
  declare id: number

  @column()
  declare systolic: number

  @column()
  declare diastolic: number
}

File: backend/health-service/src/main/java/org/padd/entity/BloodPressure.java
package org.padd.entity;

import jakarta.persistence.*;
import io.quarkus.hibernate.orm.panache.PanacheEntity;

@Entity
@Table(name = "blood_pressure")
public class BloodPressure {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "systolic")
    public int systolic;

    @Column(name = "diastolic")
    public int diastolic;


    public int getSystolic() {
        return systolic;
    }

    public void setSystolic(int systolic) {
        this.systolic = systolic;
    }

    public int getDiastolic() {
        return diastolic;
    }

    public void setDiastolic(int diastolic) {
        this.diastolic = diastolic;
    }
}
